functions/Update-DbaLocalCachedSoftware.ps1

```diff
@@ -0,0 +1,212 @@
+function Update-DbaLocalCachedSoftware {
```

Comment:
I've been thinking and I don't think this is the right name. The right name will start with `Save-Dba` but I can't think of the rest. 🤔

Reply:
What do we do? We download a branch from Github and save it to our folder for locally cached software (or replace an existing copy). Because of the "replace" I choose Update-, but Save- is also a good (and maybe better) verb. I don't know if "Software" is correct or to general? We can use "Github" and "Branch" in the name - Save-DbaGithubBranch. Maybe ask the others in #dbatools-dev?